jenkinsapi/jenkins.py

```diff
@@ -222,6 +223,10 @@ def delete_view_by_url(self, str_url):
         self.poll()
         return self
 
+    def get_label_by_url(self, str_label_url):
```

Comment:
Python projects generally do not use these Microsoft-style type prefixes on variable names. If we really want to provide guidance to users we might start using type-hints in the next major version. This variable should be called label_url.